AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -143,6 +143,8 @@ vladan.fr#@#.text-ads
 vladan.fr#@#.textAd
 vladan.fr#@#.text_ads
 !
+||*adblockerdetect.min.js^$domain=invitel.hu
```

Comment:
URL is `https://www.invitel.hu/lakossagi/assets/js/adblockerdetect.min.js`
Better use full path or `/adblockerdetect.min.js$domain=invitel.hu`
`||*` and `^` are redundant here